pages/store/index.vue

```diff
@@ -236,6 +236,7 @@ const metadataStore = useMetadataStore()
 const infiniteScrollDetectorElement = useTemplateRef<HTMLLIElement>('infiniteScrollDetector')
 const shouldLoadMore = useElementVisibility(infiniteScrollDetectorElement)
 const { handleError } = useErrorHandler()
+const { restoreScroll, saveState: savePageState, lastVisitedTag, lastVisitedQuery, clearState } = useStorePageState()
```

Comment:
Prefer not spreading properties and methods
```suggestion
const storePageState = useStorePageState()
```